### ISSUE TITLE
gpu_metrics file does not exist! always waiting

### ISSUE THREAD

**Issue by SangerY** [microsoft/nni#3077]
**Environment**:
- NNI version: 1.9
- NNI mode (local|remote|pai): local
- Client OS: ubuntu18.04
- Server OS (for remote mode only):
- Python version: 3.6.9
- PyTorch/TensorFlow version: tensorflow-gpu 1.13.1
- Is conda/virtualenv/venv used?: virtualenv
- Is running in Docker?: no 

**Log message**:
 - nnimanager.log: 
![image](https://user-images.githubusercontent.com/12136347/98531753-c2f50b80-22bb-11eb-8e98-ef7117b6045a.png)

 - dispatcher.log:
![image](https://user-images.githubusercontent.com/12136347/98531778-cc7e7380-22bb-11eb-9c77-8d5237e061a8.png)

 - nnictl stdout and stderr:
![image](https://user-images.githubusercontent.com/12136347/98531956-03548980-22bc-11eb-9e2f-e02bf3344737.png)

**What issue meet, what's expected?**:
I try to use nni in code tf-yolov3.

params = nni.get_next_parameter()
self.learn_rate_init = params['learning_rate']
![image](https://user-images.githubusercontent.com/12136347/98533662-3ac43580-22be-11eb-810f-446315f42244.png)

my search_space.json
![image](https://user-images.githubusercontent.com/12136347/98532628-ea000d00-22bc-11eb-8686-553c0b07b1f2.png)
my config.yml
![image](https://user-images.githubusercontent.com/12136347/98532678-fbe1b000-22bc-11eb-8031-18a93fd2a9e1.png)

the state is always waiting
![image](https://user-images.githubusercontent.com/12136347/98532142-43b40780-22bc-11eb-9e0d-c9b79d3ba452.png)
when I change gpuNum to zero in config.yml, the state is running, but trial metric chart shows no data. 

I find gpu_metrics in script folder
![image](https://user-images.githubusercontent.com/12136347/98533438-f20c7c80-22bd-11eb-9e7d-4bc0c3c040e4.png)

**Comment by liuzhe-lz** [comment]
Please specify `useActiveGpu: true` in your experiment configuration, if you have GUI running on the same GPU.
https://github.com/microsoft/nni/blob/v1.9/docs/en_US/Tutorial/ExperimentConfig.md#useactivegpu

**Comment by SangerY** [comment]
> Please specify `useActiveGpu: true` in your experiment configuration, if you have GUI running on the same GPU.
> https://github.com/microsoft/nni/blob/v1.9/docs/en_US/Tutorial/ExperimentConfig.md#useactivegpu

I add useActiveGpu: true in config.yml, but there is an error.
![image](https://user-images.githubusercontent.com/12136347/99036066-f5548080-25bb-11eb-8fc3-cbfaa6cad065.png)
![image](https://user-images.githubusercontent.com/12136347/99036180-259c1f00-25bc-11eb-8cba-4e50c7522de8.png)

**Comment by liuzhe-lz** [comment]
It should be:
```yaml
experimentName: ...
codeDir: ...
localConfig:
    maxTrialNumPerGpu: ...
    useActiveGpu: true
```

**Comment by SangerY** [comment]
> It should be:
> 
> ```yaml
> experimentName: ...
> codeDir: ...
> localConfig:
>   maxTrialNumPerGpu: ...
>   useActiveGpu: true
> ```

Thank you! Now it can run with gpu, but I still get nodata in chart. Is any step wrong?
![image](https://user-images.githubusercontent.com/12136347/99141148-306db700-2683-11eb-9455-6db52aed12b4.png)
my code：
![image](https://user-images.githubusercontent.com/12136347/99141142-0caa7100-2683-11eb-967a-ce831f062769.png)
![image](https://user-images.githubusercontent.com/12136347/99141130-da007880-2682-11eb-8a08-a3ed30edb2cf.png)

**Comment by liuzhe-lz** [comment]
Seems you have not invoked `nni.report_final_result()`.

**Comment by SangerY** [comment]
> Seems you have not invoked `nni.report_final_result()`.

Yes. I don't use nni.report_final_result(), just nni.report_intermediate_result()
Thank you. I will try.
